en/qgep_swmm/Check-SWMM-File.rst

```diff
@@ -93,3 +93,22 @@ Missing population densities
 ``qgep_od.catchment_area.population_density_current`` or ``qgep_od.catchment_area.population_density_planned`` are not filled.
 
 
+Subcatchment and rain Gages
+---------------------------
+If you want to attach every subcatchments to a single rain gage, edit the file with SWMM interface:
+
+- Edit > Select All
+- Edit > Group Deletion > All Rain Gages
+- Create a new raingage
+- Edit > Select All
+- Edit > Group Edit
+
+  - For objects of type: Subcatchment
+	- Edit the property: Rain Gage
```

Comment:
- Edit the property: Rain Gauge

Reply:
Same here, I will let the same spelling as in the interface